newIDE/app/src/CommandPalette/CommandManager.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+type CmdHandler = () => void | Promise<void>;
+
+export type GlobalCommand = {|
+  displayText: string,
```

Comment:
We'll need this to be a `MessageDescriptor`: `displayText: MessageDescriptor`.

So that we can register commands as translatable, using the `t` macro from js-lingui:
```
displayText: t`Open project manager`
```

and finally use `i18n._(displayText)` to do the translation in the UI

Reply:
Oh right sorry, I forgot about that! :sweat_smile: Will add it in a while